spec/support/features/idv_helper.rb

```diff
@@ -29,10 +29,11 @@ def fill_out_phone_form_fail
     fill_in :idv_phone_form_phone, with: '(703) 555-5555'
   end
 
-  def click_idv_continue(wait: false)
-    url = current_path
-    click_on t('forms.buttons.continue'), match: :first
-    expect(page).to_not have_current_path(url, wait: 10) if wait
+  def click_idv_continue
+    button = find_button(t('forms.buttons.continue'), match: :first)
+    button.click
+    # If button shows spinner when clicked, wait for it to finish.
+    expect(button).to have_no_ancestor('.spinner-button.spinner-button--spinner-active', wait: 10)
```

Comment:
`have_no_ancestor` is the funniest method name I have read in a while

But this is a great solution, thanks for adding!